container/mock_docker_client.go

```diff
@@ -0,0 +1,89 @@
+// +build ignore
+package container
+
+import (
+	"errors"
+
+	"github.com/fsouza/go-dockerclient"
+)
+
+type MockDockerClient struct {
```

Comment:
@relistan I read a bit more about this functionality and I think you're missing a blank line after that tag: "To distinguish build constraints from package documentation, a series of build constraints must be followed by a blank line." https://golang.org/pkg/go/build/#hdr-Build_Constraints

I tried to add the new line to the code, but then the tests fail to work, because `go test` treats `mock_docker_client.go` just like `go build`, apparently:
```
# github.com/nitro/sidecar-executor/container
./container_test.go:36: undefined: MockDockerClient
./container_test.go:72: undefined: MockDockerClient
./container_test.go:94: undefined: MockDockerClient
./container_test.go:125: undefined: MockDockerClient
FAIL	github.com/nitro/sidecar-executor/container [build failed]
```

So I think this approach might not work at all. Here's an example of what I meant above: https://github.com/golang/go/tree/master/src/net/internal/socktest